test/naming_tests.jl

```diff
@@ -57,3 +57,75 @@ s = savename(e; allowedtypes = (Any,), expand = ["c"])
 @test ')' ∈ s
 @test occursin("a=3", s)
 @test occursin("b=4", s)
+
+# Tests for parse_savename
+
+function test_convert(prefix::AbstractString,c;kwargs...)
+    name = savename(prefix,c;kwargs...)
+    prefix, _b, suffix = DrWatson.parse_savename(name)
+    dicts_equal(_b,c) && prefix == prefix && suffix == ""
```

Comment:
Hm, maybe `prefix == prefix ` here is a typo? (This can only be `true`, I would assume that you wanted to name the prefix of the previous line as something like `prefix2`

Reply:
Oh yes, that's nonsense :) I'll fix it